bin/download_minister.py

```diff
@@ -97,11 +139,14 @@ def download_xml_from_source():
 
     for row_en in xml_tree['en'].xpath('Minister'):
         row_fr = next(iter_fr)
+
         # set name
         name = row_en.xpath('PersonOfficialFirstName/text()')[0] + ' ' + \
                row_en.xpath('PersonOfficialLastName/text()')[0]
-        # set position code with position initials
-        position_code = (''.join([x[0].upper() for x in row_en.xpath('Title/text()')[0].split(' ')]))
+
+        # find position code in ministries dict
+        position_code = ministries_dict.keys()[ministries_dict.values().index(row_en.xpath('Title/text()')[0])]
```

Comment:
kinda nasty. Can you write the dict with keys and values swapped so you don't need to do this?